invenio_accounts/testutils.py

```diff
@@ -35,6 +35,7 @@
 
 import flask
 import flask_login
+from faker import Faker
```

Comment:
Does the `test_require` not declare dependencies needed for the tests but not installation?